tests/ci/spec_to_test_mapping.py

```diff
@@ -178,6 +178,11 @@
         '',
         'cmake-ohpc'
     ],
+    'components/dev-tools/cmake/SPECS/boost.spec': [
+        'boost',
+        '',
+        'boost-gnu12-openmpi4-ohpc'
```

Comment:
don't we need similar for `mpich` ?

Reply:
Yes, also, but the mapping is wrong. It contains cmake in the path of the spec file.

The packages tested should not be listed here, because that breaks the test. So in this case it should be empty. This is is only for additional packages which are not pulled in by RPM requires.

If you list the package here, the script will first install the results from the CI build and then for some reason DNF will downgrade the packages here to the version from the repository.

If you look at the log file you should see that the tests are not running and that boost has been downgraded.

Reply:
The downgrading cannot be seen because of the wrong mapping.

Reply:
Thanks for expanation and review, it should be:

```suggestion
        'openmpi4-gnu12-ohpc mpich-gnu12-ohpc'
```

Reply:
No need to specify those packages. They are anyway installed https://github.com/openhpc/ohpc/blob/2.x/tests/ci/setup_slurm_and_run_tests.sh#L18 and the packages should be installed as a dependency anyway.

Reply:
IIUC, L154/L159 should be empty?
https://github.com/openhpc/ohpc/blob/fb703baa96b061dcce989ff6fd284fa78bea2414/tests/ci/spec_to_test_mapping.py#L151-L160

and all deps should also be removed:

https://github.com/openhpc/ohpc/blob/fb703baa96b061dcce989ff6fd284fa78bea2414/tests/ci/spec_to_test_mapping.py#L149

right? If so, I'd like to submit a separate PR to address them.